test/units/Node-childElementCount.ts

```diff
@@ -0,0 +1,24 @@
+import { Document } from "../../deno-dom-wasm.ts";
+import { assertEquals } from "https://deno.land/std@0.85.0/testing/asserts.ts";
+
+Deno.test("Document.childElementCount", () => {
+  const document = new Document();
+  assertEquals(document.childElementCount, 0);
+});
+
+Deno.test("Element.childElementCount", () => {
+  const document = new Document();
+  const list = document.createElement("ul");
+  for (let i = 0; i < 3; i++) {
+    list.appendChild(document.createElement("li"));
+  }
+  assertEquals(list.childElementCount, 3);
+});
+
+Deno.test("Element.childElementCount with non-Element", () => {
+  const document = new Document();
+  const divider = document.createElement("div");
+  divider.appendChild(document.createElement("p"));
+  divider.appendChild(document);
```

Comment:
So... This is actually supposed to throw lol... You're not supposed to be able to append documents. What you can do instead is something like
```javascript
divider.appendChild(new Comment());
```
I'll need to push a fix for this later so it conforms to the **insert algorithm** step 7.1 "adopt node into parent's document" which should fail for actual documents as [specified here](https://dom.spec.whatwg.org/#concept-node-insert).